tests/30rooms/11leaving.pl

```diff
@@ -0,0 +1,299 @@
+multi_test "Setup a room, and have the first user leave (SPEC-216)",
```

Comment:
Since this doesn't actually check anything and merely prepares things, it probably wants to be `prepare` instead of `multi_test`.